backend/app/core/config.py

```diff
@@ -137,6 +137,14 @@ def get_s3_client(self):
             **kwargs,
         )
 
+    @computed_field
+    @property
+    def cnd_url(self) -> str | None:
+        if self.CDN_URL is not None:
+            return self.CDN_URL
+
+        return self.R2_BUCKET_NAME
```

Comment:
These aren't equivalent (CDN_URL is a navigable URL). Is this primarily to have a fallback if CDN_URL is missing?

Reply:
yep

Reply:
but yeah I see, the the fallback should be 
```suggestion
        return f"https://{self.R2_BUCKET_NAME}.s3.amazonaws.com"
```